config/config.yaml

```diff
@@ -1,38 +1,59 @@
-general:
-  metadata_file: "metadata.tsv"
-  threads: 8
+threads: 1
 
 bowtie2:
-  idx: 'data/external/ref/idx/bowtie2/bowtie2'
-  maxins: 2000
+  idx: 'data/external/ref/idx/bowtie2/index'
```

Comment:
Bowtie2 index now more in line with the standard from get_ref_genomes. Why is the maxins now as a string?